### DISCUSSION TITLE
lesson 6 - foundry simple storage

### DISCUSSION THREAD

**Discussion by callyongg** [Cyfrin/foundry-full-course-cu#1979] [General]
Can anyone help me with this error :
```
No files changed, compilation skipped
Error: 
error sending request for url (http://127.0.0.1:7545/)

Context:
- Error #0: error sending request for url (http://127.0.0.1:7545/)
- Error #1: client error (Connect)
- Error #2: tcp connect error: Connection refused (os error 111)
- Error #3: Connection refused (os error 111)
```
I got this while I want to deploy using forge create SimpleStorage --rpc-url http://127.0.0.1:7545
I got stuck here, please help me:)

**Comment by EngrPips** [comment]
Hello @yongcen, Please ensure you run `anvil` on a separate terminal, keep it running, and then retry the command.

**Reply by callyongg** [reply]
Thankyou for helping me @EngrPips it works

**Reply by EngrPips** [reply]
Awesome, Keep crushing it!
